var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -463,6 +484,8 @@ def plugin_cmake_variant(plugin_name, spack_variant):
             plugin_cmake_variant("TGEO", "tgeo"),
             example_cmake_variant("TBB", "tbb", "USE"),
             cmake_variant(unit_tests_label, "unit_tests"),
+            plugin_cmake_variant("TRACCC", "traccc"),
+            self.define_from_variant("ACTS_CUSTOM_SCALARTYPE", "scalar"),
```

Comment:
Minor. Move up so it is alphabetically sorted based on `scalar`.

Reply:
Done!